ppx/jsx-ppx/tyxml_ppx_jsx.ml

```diff
@@ -0,0 +1,96 @@
+open Ast_mapper
+open Parsetree
+open Ast_helper
+open Asttypes
+
+open Tyxml_ppx_common
+
+let make_html_attr_name name =
+  let name =
+    match name with
+    | "class_" -> "class"
+    | "className" -> "class"
+    | "htmlFor" -> "for"
+    | "type_" -> "type"
+    | "to_" -> "to"
+    | "open_" -> "open"
+    | "begin_" -> "begin"
+    | "end_" -> "end"
+    | "in_" -> "in"
+    | name -> name
```

Comment:
Where does that list come from ? Is it just "standard" practice in JSX ? Otherwise, we might need the HTML/CSS<->JS full translation (which is implemented somewhere in jsoo).

Reply:
https://github.com/cxa/ppx_bsx/blob/master/src/ppx_bsx_lib.ml#L16 i basically took a mix from this one and this one https://github.com/bikallem/jsx-ssr/blob/8bf3101d1b6ff71c96c3a1f8462cd9e137dfc8b2/lib/Jsx.re#L62. Unfortunately there does not seem to be any official convention, just loose common way of renaming :/